### PR TITLE
Nbft ignore ifname rhel 9

### PR DESCRIPTION
**Pull request by rvykydal**
Port of https://github.com/rhinstaller/anaconda/pull/5771

### CODE REVIEW

**Review by tbzatek — APPROVED** [review]
Tested on `RHEL-9.5.0-20240820.2`, confirming everything works as expected!

File: pyanaconda/modules/network/initialization.py [hunk 1]
```diff
@@ -232,6 +232,10 @@ def _run(self, nm_client):
 
             iface = device.get_iface()
 
+            if is_nbft_device(iface or ""):
```
Comment (jkonecny12) [inline pyanaconda/modules/network/initialization.py:239]: This totally work, I'm just thinking that:
```python
if iface and is_nbft_device(iface):
```
wouldn't be better to read.

No need to update the code, just saying for future I guess :D :D

**Review by jkonecny12 — APPROVED** [review]
Looks good to me. Thanks a lot!

**Pull request merged**